README.md

```diff
@@ -110,7 +110,22 @@ All metrics are exported as gauges.
 
 In addition `cloudwatch_exporter_scrape_error` will be non-zero if an error
 occurred during the scrape, and `cloudwatch_exporter_scrape_duration_seconds`
-contains the duration of that scrape.
+contains the duration of that scrape. `cloudwatch_exporter_build_info` will be
+non-zero if an error occurred scraping the build info.
+
+### Build Info Metric (New)
+`cloudwatch_exporter_build_info` is a new default cloudwatch exporter metric that contains the current
```

Comment:
```suggestion
### Build Info Metric

`cloudwatch_exporter_build_info` is a default cloudwatch exporter metric that contains the current
```

The README describes the current state, we don't have to say "new" here (and it would be out of date one release later anyway).